server/src/test/java/io/spine/server/model/given/filter/ModSplitEventAggregate.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.model.given.filter;
+
+import io.spine.core.Where;
+import io.spine.server.aggregate.Aggregate;
+import io.spine.server.aggregate.Apply;
+import io.spine.server.command.Assign;
+import io.spine.test.model.ModCreateProject;
+import io.spine.test.model.ModProjectCreated;
+import io.spine.test.model.filter.ModProject;
+
+/**
+ * An invalid aggregate declaration. Appliers cannot have {@link Where} filers.
+ */
+public final class ModSplitEventAggregate
+        extends Aggregate<String, ModProject, ModProject.Builder> {
+
+    private static final String FANCY_ID = "1";
+    private static final String INTERNAL_FANCY_ID = "fancy project";
+
+    @Assign
+    ModProjectCreated handle(ModCreateProject cmd) {
+        return ModProjectCreated
+                .newBuilder()
+                .setId(cmd.getId())
+                .build();
+    }
+
+    @Apply
+    private void onFancy(@Where(field = "id", equals = INTERNAL_FANCY_ID) ModProjectCreated e) {
+        builder().setProjectId(FANCY_ID);
+    }
+
+    @Apply
+    private void on(ModProjectCreated e) {
```

Comment:
I am interested in knowing what happens if

a) There will be just two appliers like these:

```java
@Apply
private void hanky(@Where(field = "id", equals = "hanky") ModProjectCreated e) { ... }

@Apply
private void panky(@Where(field = "id", equals = "panky") ModProjectCreated e) { ... }

// NOTE: no "general" option.
```

Will it blend?

or b) — just two appliers like this one

```java
@Apply
private void one(@Where(field = "id", equals = "copy-paste") ModProjectCreated e) { ... }

```java
@Apply
private void another(@Where(field = "id", equals = "copy-paste") ModProjectCreated e) { ... }

// NOTE: copy-pasted methods with different names, but the same `equals` part!
```

Maybe, this one blends?

Reply:
Same as above.

Al